pkg/pop/msal.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package pop
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/AzureAD/microsoft-authentication-library-for-go/apps/confidential"
+	"github.com/AzureAD/microsoft-authentication-library-for-go/apps/public"
+)
+
+// acquires a PoP token using MSAL's interactive login flow. Requires user to authenticate via browser
+func AcquirePoPTokenInteractive(
```

Comment:
why does it not have `options *azcore.ClientOptions` like below?

Reply:
Added the client options here